cli/module_registry.go

```diff
@@ -242,6 +346,23 @@ func resolveOrg(client *AppClient, orgID, publicNamespace string) (*apppb.Organi
 	return org, nil
 }
 
+func loadManifest(manifestPath string) (ModuleManifest, error) {
+	if _, err := os.Stat(manifestPath); err != nil {
+		return ModuleManifest{}, errors.Wrapf(err, "Cannot find %s", manifestPath)
+	}
+
+	//nolint:gosec
+	manifestBytes, err := os.ReadFile(manifestPath)
```

Comment:
readFile will return an error if the file doesn't exist so the stat above isn't necessary. If you want to do this to return for a specific error you can do this: 

```suggestion
	manifestBytes, err := os.ReadFile(manifestPath)
	if err != nil {
	     if errors.Is(err, fs.ErrNoExist) {
	          return ModuleManifest{}, errors.Wrapf(err, "Cannot find %s", manifestPath)
	     }
	     return ModuleManifest{}, err
	}
	
```

Reply:
That is so much better. Good idea.